.stylelintrc

```diff
@@ -22,6 +22,7 @@
     "unit-no-unknown": null,
     "no-descending-specificity": null,
     "value-list-max-empty-lines": null,
-    "selector-type-no-unknown": null
+    "selector-type-no-unknown": null,
+    "selector-class-pattern": null
```

Comment:
CodeFactor always complains the classname or id need to be `kebab-case` which is quite annoying. Disable this rule.
<img width="1230" alt="Screenshot 2024-07-01 at 5 11 17 PM" src="https://github.com/longhorn/longhorn-ui/assets/5744158/836b54fb-f624-4a6c-9b35-7e1d7953cba2">